Description: Don't build js libraries from third parties
 These 3rd party libraries are getting excluded from the source through
 repacking since we instead include those from the debian packages that already
 exists for them. The build shouldn't try and process the non-existent files.
Author: Gabriel Filion <gabster@lelutin.ca>
Forwarded: not-needed
Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)

--- smokeping-2.8.2+ds.orig/htdocs/Makefile.am
+++ smokeping-2.8.2+ds/htdocs/Makefile.am
@@ -16,21 +16,6 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
 JS = js/smokeping.js \
-    js/prototype.js \
-    js/cropper/cropper.js \
-    js/cropper/licence.txt \
-    js/cropper/marqueeVert.gif \
-    js/cropper/cropper.uncompressed.js \
-    js/cropper/cropper.css \
-    js/cropper/marqueeHoriz.gif \
-    js/scriptaculous/controls.js \
-    js/scriptaculous/slider.js \
-    js/scriptaculous/sound.js \
-    js/scriptaculous/effects.js \
-    js/scriptaculous/unittest.js \
-    js/scriptaculous/builder.js \
-    js/scriptaculous/scriptaculous.js \
-    js/scriptaculous/dragdrop.js \
     css/smokeping-print.css \
     css/smokeping-screen.css
 
